### ISSUE TITLE
[TW-1066] Cannot change project on recurring tasks

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#1101]
_John Florian on 2010-06-13T11:24:59Z says:_ 

 I was trying to reorganize my plethora of tasks into a new project structure.  I have a recurring task under project X and want to change it project Y, but I get the following error:

(at)You cannot remove the recurrence from a recurring task.@

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2010-06-13T11:24:59Z
Modified: 2014-02-09T02:16:26Z
```

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2010-06-13T11:31:40Z says:_ 

 That code was recently touched.  Thanks John.

**Comment by taskwarrior** [comment]
_John Florian on 2010-06-13T11:59:26Z says:_ 

 Paul Beckingham wrote:
> That code was recently touched.  Thanks John.

I should mention that this is 1.8.5 -- I know I'm running old code, but it's the latest in the Fedora 12 repositories and I really don't have time to be manually installing it on so many systems.  (I guess that might be a "downside" to a CLI -- I expect to use it everywhere.)

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2010-06-13T12:13:30Z says:_ 

 Fixed in 1.9.1.
